charts/hivemq-platform/tests/configuration/hivemq_control_center_listeners_configuration_test.yaml

```diff
@@ -0,0 +1,570 @@
+suite: HiveMQ Platform Configuration - Control Center listeners tests
```

Comment:
Should we have a failing test when someone tries to use `hivemqListenerName` with `type: control-center`?

Reply:
Nice catch!